client/store/index.js

```diff
@@ -18,8 +18,10 @@ const store = createStore(reducer, undefined,
   )
 )
 
-if (module.hot) {
-  module.hot.accept('../reducers', () => {
+const hotLoader = module.hot
```

Comment:
You can put this back the way it ways if you upgrade the latest eslint-config-zeal